.github/workflows/test-ui-e2e-only-desktop.yml

```diff
@@ -103,7 +103,7 @@ jobs:
         id: tests
         run: |
           export COINAPPS=$PWD/coin-apps
-          if [ "$ENABLE_BROADCAST" = true ]; then export ENABLE_TRANSACTION_BROADCAST=1; fi
+          if [ "$ENABLE_BROADCAST" = true || github.event_name == 'schedule']; then export ENABLE_TRANSACTION_BROADCAST=1; fi
```

Comment:
does it make sense to broadcast every night ? (especially if mainnet are in use)

Reply:
For now I only have a send test for devnet (sepolia)

Reply:
this is not a correct condition, please fix it